### DISCUSSION TITLE
Welcome to Django Bridge Discussions!

### DISCUSSION THREAD

**Discussion by kaedroho** [kaedroho/django-bridge#2] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ChrisB49** [comment]
Does this library support any React Suspense functionality? Have a Data visualization page I'm thinking of using this module for on my Django Server, however if it doesn't support Suspense/Streaming that would be problematic

**Reply by kaedroho** [reply]
I think this should work, but I haven't tried it.

Django Render will render your page component, and that could contain a `<Suspense>` block that pulls in extra data from a different source (which could be a REST API provided by your Django server).

**Comment by KungaCoding** [comment]
maaan i just came here to share how amazing you are for building this.  I've been programming for only a few years now and this is game changing for me.  Thank you thank you thank you! Love the work you're doing!

**Reply by kaedroho** [reply]
Thanks a lot for taking the time to post this @KungaCoding!

**Comment by sandip-sadhukhan** [comment]
@kaedroho Thanks for creating this library, seems promising! It will fill the gap between Django and React and we can use best of both worlds without creating APIs etc.

One suggestion from my side, currently on each page load we are fetching all the client side javascript files which is not optimized for example, the CMS you build, if we go to the login page it will fetch all the javascript chunks like code for dashboard components etc which is not required on login page. Can we expect code spliting feature in django-bridge?

**Reply by kaedroho** [reply]
Thanks @sandip-sadhukhan!

It should work with the code-splitting feature built into React/Vite. I'll look at adding this to the CMS demo (the block note editor it uses is quite heavy so would be a good candidate)
